app/src/main/java/com/infomaniak/mail/ui/LaunchActivity.kt

```diff
@@ -135,4 +140,43 @@ class LaunchActivity : AppCompatActivity() {
             }
         }
     }
+
+
+    private fun handleShortcuts(isUserDisconnected: Boolean): Boolean {
+        intent.extras?.getString(SHORTCUTS_TAG)?.let {
+            return runByShortcut(it, isUserDisconnected)
+        }
+        return false
```

Comment:
return intent.extras?.getString(SHORTCUTS_TAG)?.let { runByShortcut(it, isUserDisconnected) } ?: false

But I think you can merge this function with the runByShortcut (and funbd a better name)